### ISSUE TITLE
Show multimonth sub announcements

### ISSUE THREAD

**Issue by deanpcmad** [chatty/chatty#496]
**Is your feature request related to a problem? Please describe.**

When someone subscribes for a multi-month sub, it doesn't show in chatty

**Describe the solution you'd like**

An addition to the subscriber message saying how many months

**Additional context**

Twitch use `msg_param_multimonth_duration` for this (at least how my app saves it, it may not be underscores). Looks like they also show that for most subs, but for non multimonth subs it shows 0

**Comment by tduva** [comment]
Can you provide an example of what a multimonth sub looks like in Twitch Chat?

**Comment by deanpcmad** [comment]
Finally managed to get a screenshot:

![](https://files.deanpcmad.com/2023/OYJJz/KOgbIOlbFs.png)

**Comment by tduva** [comment]
I wonder under what circumstances it shows up like that, because when I subbed for several months in advance it didn't. Makes it a bit difficult to test, but I added something know, which will be in the next beta.

**Comment by deanpcmad** [comment]
Yeah I don't think it shows to the person who subscribed, but it shows that message the others in chat.

Also, for resubs, it shows like so:

![](https://files.deanpcmad.com/2023/nbaiW/kAZwrCUZ6h.png)

**Comment by SputNikPlop** [comment]
I hope these help.
![Screen Shot 2023-06-23 at 9 57 40 AM copy](https://github.com/chatty/chatty/assets/100245448/ab4e
![Screen Shot 2023-06-30 at 11 34 49 AM](https://github.com/chatty/chatty/assets/100245448/a1234743-3a35-4115-b9bf-4265d35d227b)
e5b2-a248-4368-9ed1-075c956bd783)
